src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -7657,6 +8165,199 @@ subroutine scale_obs_tskin_zscore( N_catd, tile_coord,   &
     
   end subroutine scale_obs_tskin_zscore
 
+ 
+  ! *****************************************************************
+  
+  subroutine scale_obs_sfmc_zscore( N_catd, tile_coord,         &
```

Comment:
A reminder that we need to deal with unphysical scaled soil moisture values -- could add as a new check in get_obs_pred(), but must be separate from qc_model(), which only disables obs by setting obs_pred to no-data.  would need to add functionality that computes obs-space porosity and then cuts back scaled soil moisture obs to obs-space porosity at the high end and to zero at the low end